specs/test_2.js

```diff
@@ -0,0 +1,30 @@
+let MainPage = require("../pages/main.page");
+let LoginPage = require("../pages/login.page");
+let RegistrationPage = require("../pages/registration.page");
+
+describe('Protractor Demo App', () => {
```

Comment:
Please update suite name, you can have few suites like registration suite, login suite ...